src/EventStore.php

```diff
@@ -279,6 +299,36 @@ private function getStreamUrl(string $streamName): string
         return sprintf('%s/streams/%s', $this->url, $streamName);
     }
 
+    /**
+     * @param string $url
+     *
+     * @return string
+     */
+    private function addCredentialsToUrl(string $url): string
+    {
+        $parts = parse_url($url);
+        $parts['user'] = $this->credentials['user'];
+        $parts['pass'] = $this->credentials['pass'];
+        $newUrl = http_build_query($parts);
+        var_dump($newUrl);
+        exit;
```

Comment:
I think you forgot to remove this

Reply:
Oops, I'm sorry I have a read-through to do, concerning version binding among others (like coverage in CI...). I didn't thought you would answer that fast relatively to the date of your last commit.

Thank you very much for giving attention to my work that fast. I'll remove some remaining comments and others like this var_dump before the night